sycl/doc/PreprocessorMacros.md

```diff
@@ -38,16 +38,18 @@ This file describes macros that have effect on SYCL compiler and run-time.
 
   Disables a message which warns about unsupported C++ version.
 
-- **SYCL_DISABLE_FALLBACK_ASSERT**
-
-  Defining this macro eliminates some overhead that is associated with
-  submitting kernels that call `assert()`. When this macro is defined, the logic
-  for detecting assertion failures in kernels is disabled, so a failed assert
-  will not cause a message to be printed and will not cause the program to
-  abort. However, this macro only affects kernels that are submitted to devices
-  that do **not** have native support for `assert()` because devices with native
-  support do not impose any extra overhead. One can check to see if a device has
-  native support for `assert()` via `aspect::ext_oneapi_native_assert`.
+- **SYCL_ENABLE_FALLBACK_ASSERT**
```

Comment:
Should we have it in the form SYCL_FALLBACK_ASSERT=0/1 ?
This way we won't need to change name back and forth.

I'm kind of surprised this patch is hanging for so long... does it mean we don't have a problems with having it ON by default? Why are we disabling by default?

Reply:
Renamed the macro.